### ISSUE TITLE
Getting end timestamp in result(s)

### ISSUE THREAD

**Issue by rohithkodali** [k2-fsa/sherpa-onnx#982]
Is there any possibility to get start and end timestamp both for every token, currently we are getting only start time of every token.

A scenario that we face in general is, we have multiple speech samples where there is a long silence (>0.5 seconds to 3 seconds) as we have only the start time, the particular word is kind of spoken for 3 seconds which is not a proper scenario when we check the duration analysis.

Another question, is it possible to get logits in the result(s) along with timestamps so that we can apply some other algorithms on them?

**Comment by csukuangfj** [comment]
Please see
https://github.com/k2-fsa/sherpa-onnx/pull/989

Currently, we can only get stop timestamp of each token for CTC models.

**Comment by csukuangfj** [comment]
> Another question, is it possible to get logits in the result(s) along with timestamps so that we can apply some other algorithms on them?

Please see

https://github.com/k2-fsa/sherpa-onnx/blob/09efe548084a5b216ebef0b725555a0cd4b76027/sherpa-onnx/csrc/online-recognizer.h#L44
